homepage/blocks/homepage-brick/homepage-brick.js

```diff
@@ -159,6 +115,21 @@ export default async function init(el) {
     const [highlight, ...tail] = rows;
     highlight.classList.add('highlight-row');
     rows = tail;
+  } else {
+    let [head, ...tail] = rows;
+    const classList = el.className.split(' ');
+    classList.splice(1, 0, 'click');
+    el.className = classList.join(' ');
```

Comment:
Is there a reason why we want to add `click` as second classname instead simply add 'click' for the element?
like:
```suggestion
    el.classList.add('click');
```

Reply:
there WAS a good reason.  But we updated the way the new analytics taxonomy will work earlier this week, and it makes this code unnecessary.